specification/index.md

```diff
@@ -17,5 +17,5 @@ The following pages introduce the syntax and structure of an OpenAPI document, i
 - [Content of Message Bodies](content): `content` and `schema`.
 - [Parameters and Payload of an Operation](parameters): `parameters` and `requestBody`.
 - [Reusing Descriptions](components): `components` and `$ref`.
-- [Providing Documentation and Examples](docs): `example` and `examples`.
+- [Providing Documentation and Examples](docs): `description` and `examples`.
```

Comment:
I'm not sure this was a typo. `example` and `examples` are two different things.

Reply:
The 'description' is missing there. Can be one of:
- 'description' and 'example' (for simplicity)
- 'description', 'example' and 'examples' (for accuracy)